Kickstarter-iOS/Views/Controllers/PledgeShippingLocationViewController.swift

```diff
@@ -117,6 +112,24 @@ final class PledgeShippingLocationViewController: UIViewController {
     self.shimmerLoadingView.rac.hidden = self.viewModel.outputs.shimmerLoadingViewIsHidden
     self.shippingLocationButton.rac.title = self.viewModel.outputs.shippingLocationButtonTitle
 
+    /**
```

Comment:
Needed to do this a bit more aggressively for the header to correctly resize 🤷‍♂️